client/templates/course/course.less

```diff
@@ -0,0 +1,22 @@
+.course-page-content {
+    margin-top: 3em;
+}
+
+.course-page-content > p {
+    font-size: 1.6rem;
+}
+
+.component-image {
+    margin-top: 1em;
+    margin-bottom: 1em;
+}
+
+.image-descr {
```

Comment:
Also, consider renaming the class to "image-caption" in order to align with the schema.org [ImageObject schema](http://schema.org/caption).